Automattic/wp-calypso
# Update related post demo images

## Changes proposed in this Pull Request

This PR simply updates the demo images shown on Settings > Traffic > Related Posts > (switch show related content after posts) (switch show a thumbnail imaged where available)

Props to @johnmaeda for coming up with these three new images.

### Before
![](https://d1jfzjx68gj8xs.cloudfront.net/items/063q3X3S2s2E3N383B3C/Screen%20Shot%202019-03-16%20at%2010.46.50%20AM.png)

### After
![](https://d1jfzjx68gj8xs.cloudfront.net/items/440m1j430e0w2U2I1l20/Screen%20Shot%202019-03-16%20at%2010.55.49%20AM.png)

## Testing instructions

- Check out this branch
- Compile changes
- View http://calypso.localhost:3000/settings/traffic/{YOUR-DOMAIN}

Fixes https://trello.com/c/ryDVvl2C/105-antique-images-for-related-posts
